### ISSUE TITLE
[Bug] Cover images sometimes don't work with Apple Music on Mac

### ISSUE THREAD

**Issue by ungive** [ungive/discord-music-presence#77]
The logs contain this error:

```
presence 2024-11-07T12:14:03.959Z ERROR Using cover image from player failed: Failed to convert image for proxy: Conversion result is empty type=image/jpeg size=588493
```

With help of some of our Discord members it turned out Apple Music seems to be reporting cover images as JPEG images, while in fact they are PNG images:

![image](https://github.com/user-attachments/assets/b973e26e-6ac8-442a-abd5-aa0f85cf7d0a)

The `image-jpeg` text is derived from the MIME type that is reported by the operating system (`image/jpeg`). Either Apple Music believes that's a JPEG or macOS misidentifies the image as JPEG and reports it wrong. I'll confirm later it's not a bug in Music Presence.

**Comment by ungive** [comment]
If you have this issue, there's a build in the Discord that fixes it: https://discord.gg/musicpresence

This will also be fixed in the next version of Music Presence.

**Comment by ungive** [comment]
Fixed in 2.2.6
